### PR TITLE
Add post-install image as dependency of conformance tests

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

**Review by aliok — APPROVED** [review]
/hold

LGTM, but are we using it? Unhold if so

File: ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.3__49.yaml
Comment (matzew) [inline ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.3__49.yaml:279]: should we call it more like `knative-eventing-kafka-controller-post-install` ?

No need to do it now - but general fruit for thought

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by pierDipi** [comment]
> /hold
> 
> LGTM, but are we using it? Unhold if so

Yes, I'm using it in the new PR https://github.com/openshift-knative/eventing-kafka-broker/pull/221/files#diff-8d4780ca239f8b787f8e8831ea33becadb5238ceabc0b59b55e8ce4d7c679e39

**Pull request merged**
